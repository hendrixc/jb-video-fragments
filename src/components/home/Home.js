import React, { Component } from 'react';
import Header from '../header/Header';
import Content from '../content/Content';
import Menu from '../menu/Menu';
import './home.css';

class Home extends Component {
  render() {
    return (
      <div className="home">
        <Header />
        <div className="content">
          <Menu />
          <Content />
        </div>
      </div>
    );
  }
}

export default Home;
