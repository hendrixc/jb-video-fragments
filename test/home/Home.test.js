import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../src/components/home/Home';

jest.mock('../../src/components/video/Video', () => 'Video');
jest.mock('../../src/components/clips/ClipList', () => 'ClipList');

it('renders an home component into the Home as snapshot', () => {
  const TextInputComponent = renderer.create(<Home />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
