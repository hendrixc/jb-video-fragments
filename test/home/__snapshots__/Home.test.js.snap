// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an home component into the Home as snapshot 1`] = `
<div
  className="home"
>
  <div
    className="Header"
  >
    <div
      className="AppBar"
    >
      <header
        className="mdc-top-app-bar"
        style={Object {}}
      >
        <div
          className="mdc-top-app-bar__row"
        >
          <section
            className="mdc-top-app-bar__section"
          >
            <i
              className="rmwc-icon material-icons mdc-top-app-bar__navigation-icon"
              data-mdc-ripple-is-unbounded={true}
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              style={Object {}}
            >
              menu
            </i>
            <div
              className="mdc-top-app-bar__title"
            >
              Video Fragments Tube
            </div>
          </section>
          <section
            className="mdc-top-app-bar__section mdc-top-app-bar__section--align-end"
          >
            <i
              className="rmwc-icon material-icons mdc-top-app-bar__navigation-icon"
              data-mdc-ripple-is-unbounded={true}
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              style={Object {}}
            >
              favorite
            </i>
          </section>
        </div>
      </header>
    </div>
  </div>
  <div
    className="content"
  >
    <div
      className="menu"
    >
      <div
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <aside
          className="mdc-drawer"
        >
          <div
            className="mdc-drawer__header"
          >
            <div
              className="mdc-drawer__title"
            >
              DrawerHeader
            </div>
            <div
              className="mdc-drawer__subtitle"
            >
              Subtitle
            </div>
          </div>
          <div
            className="mdc-drawer__content"
          >
            <div
              className="mdc-list"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
            >
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Cookies
              </div>
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Pizza
              </div>
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Icecream
              </div>
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Icecream
              </div>
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Icecream
              </div>
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Icecream
              </div>
              <div
                className="mdc-list-item"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={Object {}}
                tabIndex={0}
              >
                Icecream
              </div>
            </div>
          </div>
        </aside>
      </div>
    </div>
    <div
      className="content"
    >
      asdasd
    </div>
  </div>
</div>
`;
